Ginger/Ginger/Dictionaries/SetTerminologyTypePage.xaml.cs

```diff
@@ -68,6 +68,7 @@ private void okBtn_Click(object sender, RoutedEventArgs e)
         {
             if (App.UserProfile.TerminologyDictionaryType != mInitialType)
             {
+                GingerTerminology.TERMINOLOGY_TYPE = mInitialType;
                 Reporter.ToUser(eUserMsgKeys.SettingsChangeRequireRestart);
```

Comment:
SetTerminologyTypePage should be deleted, it's not been used anymore

Reply:
Fixed